cmd/agent/dist/conf.d/openmetrics.d/conf.yaml.default

```diff
@@ -0,0 +1,10 @@
+instances:
+  -
+    openmetrics_endpoint: http://localhost:5000/telemetry
+    namespace: datadog.agent
```

Comment:
This was chosen somewhat arbitrarily, and I'm not sure if there's a better choice.